kitsune/search/es_utils.py

```diff
@@ -32,10 +32,15 @@ class UntypedS(object):
 def F(*args, **kwargs):
     pass
 
+
+def get_es():
```

Comment:
We should at least have this print a warning or something. It still needs to go away entirely.

Reply:
Maybe can add a comment with `TODO:`